app/src/main/java/com/github/se/cyrcle/ui/gambling/GamblingScreen.kt

```diff
@@ -281,20 +330,98 @@ fun GamblingScreen(navigationActions: NavigationActions, userViewModel: UserView
             contentAlignment = Alignment.Center) {
               var wheelSpinFunction by remember { mutableStateOf<(() -> Unit)?>(null) }
 
-              Text(
-                  text = stringResource(R.string.gambling_screen_coins_display, coins ?: 0),
-                  modifier =
-                      Modifier.align(Alignment.TopCenter)
-                          .padding(top = 16.dp)
-                          .testTag("coin_display"),
-                  style = MaterialTheme.typography.headlineMedium)
+              Column(
+                  horizontalAlignment = Alignment.CenterHorizontally,
+                  modifier = Modifier.align(Alignment.TopCenter)) {
+                    Row(
+                        modifier = Modifier.fillMaxWidth(0.8f).padding(bottom = 4.dp),
+                        horizontalArrangement = Arrangement.SpaceBetween) {
+                          Text(
+                              text = stringResource(R.string.level_display, displayedLevel),
+                              style = MaterialTheme.typography.bodyMedium,
+                              modifier = Modifier.testTag("current_level_text"))
+                          Text(
+                              text = stringResource(R.string.level_display, (displayedLevel + 1)),
+                              style = MaterialTheme.typography.bodyMedium,
+                              modifier = Modifier.testTag("next_level_text"))
+                        }
+
+                    LinearProgressIndicator(
+                        progress = { animatedProgress.value },
+                        modifier =
+                            Modifier.fillMaxWidth(0.8f).height(8.dp).testTag("level_progress_bar"),
```

Comment:
by camel case you mean LevelProgressBar instead of snake level_progress_bar ?

Reply:
Exactly right!

Reply:
i changed them all one by one in GamblingScreen and in GamblingScreenTest. Took me quite a while because chatgpt is too dumb to do that repetitive task... And i thought AI was gonna steal my job...